.github/workflows/pytest-core-mpi.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     env:
       DEVITO_LANGUAGE: "openmp"
       DEVITO_ARCH: "gcc-9"
+      OMP_NUM_THREADS: "1"
```

Comment:
?

Reply:
Avoid unnecessary thread concurency sonce this is mpi

Reply:
the MPI+openmp is tested with the mpi examples but this is a small 2 core machine that is already oversubscribed with MPI, adding threads only make things slower for no reason

Reply:
fair enough

for another day: use some kind of self-hosted runner for MPI+OpenMP build? if agree, maybe add to issue list